apps/cicognara-static/_site/catalogo/itempages/1.html

```diff
@@ -16,15 +16,10 @@
       <main id="main">
          <div class="container">
             <section class="catalogo-item-header">
-               <h1>1.
-                  
-                  	      AGINCOURT (d’) Seroux, Histoire de l’Art
```

Comment:
This change is a little worrying — the entry being deleted is definitely correct, and the substituted text below appears much further down as a child of another entry — maybe an XPath issue?

Reply:
very worrying. I also don't know what the empty TEI files are doing there.  I'll review the generation scripts again